core/src/main/scala/cats/data/OneAnd.scala

```diff
@@ -104,7 +111,7 @@ private[data] sealed trait OneAndInstances extends OneAndLowPriority2 {
     }
 
   implicit def oneAndSemigroup[F[_]: MonadCombine, A]: Semigroup[OneAnd[F, A]] =
-    oneAndSemigroupK.algebra
+    oneAndSemigroupK(MonadCombine[F]).algebra
```

Comment:
Sorry, this is nitpicky, but could you please change this to `oneAndSemigroupK[F].algebra`? That will save a `MonadCombine.apply` call.